### PR TITLE
update developer setup info

### PR DESCRIPTION
**Pull request by ChristophWurst**

### CODE REVIEW

**Comment by ChristophWurst** [comment]
@jancborchardt @Gomez @irgendwie @enoch85 review please

**Comment by enoch85** [comment]
Will this break my nightly script?

**Comment by ChristophWurst** [comment]
> Will this break my nightly script?

That depends on how you generate nightlies. Do you use the Makefile?

**Comment by enoch85** [comment]
@ChristophWurst Yes: https://gist.github.com/enoch85/c683de8ef534bdd81946

**Comment by ChristophWurst** [comment]
> @ChristophWurst Yes: https://gist.github.com/enoch85/c683de8ef534bdd81946

I don't see any problems there. Should still work

**Comment by enoch85** [comment]
Then :+1: from me

File: README.md [hunk 1]
```diff
@@ -72,13 +72,15 @@ namespace inbox {
 
 ## Developer setup info
 
-Just clone this repo into your apps directory ([ownCloud core installation needed](https://doc.owncloud.org/server/8.1/developer_manual/general/devenv.html)). Additionally you need Composer to install PHP dependencies and npm to get the Javascript dependencies – run this from inside the mail folder:
+Just clone this repo into your apps directory ([ownCloud core installation needed](https://doc.owncloud.org/server/8.2/developer_manual/general/devenv.html)). Additionally,  [NPM](https://docs.npmjs.com/getting-started/installing-node) is needed for installing JavaScript dependencies. On Debian and Ubuntu Systems it can be installed with the following command:
```
Comment (jancborchardt) [inline README.md]: »npm« is always lowercase (below as well). ;)

File: README.md [hunk 2]
```diff
@@ -72,13 +72,15 @@ namespace inbox {
...
 sudo install node nodejs-legacy
-curl -sS https://getcomposer.org/installer | php
-php composer.phar install
 ```
-Debian/Ubuntu people, just execute `install_ubuntu.sh`
+Once NPM is installed, PHP and JavaScript dependencies can be installed by running
+```bash
+make dev-setup
+make optimize-js
+```
```
Comment (jancborchardt) [inline README.md]: Can we just have that in one command?

**Comment by jancborchardt** [comment]
Except the comments I made inline it looks good and the commands seem to work. :)

**Comment by itheiss** [comment]
LGTM :+1:

**Pull request merged**